### PR TITLE
Individual pages

### PR DESCRIPTION
**Pull request by ramirezg**
- creating pretty searchable tables for spaces and orgs
- setting up system for efficiently rendering individual pages for orgs. 
tldr - org data from the `/v2/organizations` endpoint is only loaded on entry, if prompted, or if an error occurs. This should keep costly api calls down especially when we create space/app/service pages.

### CODE REVIEW

File: static/js/app/cloudfoundry.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 (function() {
     // CloudFoundry Service
-    var cloudfoundry = function($http, $location) {
+    angular.module('cfconsole').service('$cloudfoundry', function($http, $location, $log) {
```
Comment (jcscottiii) [inline static/js/app/cloudfoundry.js]: change this to cfdeck

File: static/index.html [hunk 2]
```diff
@@ -22,7 +22,7 @@
 			<div class="site-wrapper-inner">
 				<div class="container">
 				<!-- Where our view will load -->
-	    		<div ng-view></div>
+	    		<div ng-view ng-controller="DashboardCtrl"></div>
```
Comment (jcscottiii) [inline static/index.html]: Why are you putting this into the base file and not the partials/index.html?

File: static/js/app/controllers.js [hunk 3]
```diff
@@ -11,61 +11,62 @@
...
-                });
+    app.controller('OrgCtrl', function($scope, $cloudfoundry, $location, $routeParams) {
+        // Display Space
+        $scope.showSpace = function(space) {
+            $location.path($location.path() + '/spaces/' + space.guid)
+        };
+        // Render the org information on the page
+        var renderOrg = function(orgData) {
+            if (orgData['code'] == 30003) {
+                $scope.activeOrg = "404";
```
Comment (jcscottiii) [inline static/js/app/controllers.js]: Comment as to what the 30003 is and why we should set activeOrg to 404 and what does sending a 404 do.

Comment (jcscottiii) [inline static/js/app/controllers.js]: guessing it has to do with https://github.com/18F/cf-deck/pull/30/files#diff-60bb1f6b95d957acd2a186fed9e2db20R3

File: static/js/app/partials/organizations.html [hunk 4]
```diff
@@ -1,12 +1,75 @@
...
-    <div ng-switch-default>
-        <ng-include src="'js/app/partials/spaces.html'"></ng-include>
+    <div ng-switch-when="404">
+        <div class="page-header">
+            <h3 class="text-center">Sorry, we couldn't find that org</h3>
+        </div>
+    </div>
+
+    <div ng-switch-when="undefined">
+        <!-- show nothing while loading -->
```
Comment (jcscottiii) [inline static/js/app/partials/organizations.html]: In the future, we could do a loading spinner. But for now, put something like "Loading..."

File: static/js/app/partials/organizations.html [hunk 5]
```diff
@@ -1,12 +1,75 @@
...
+                        <td>{{space.name}}</td>
+                        <td>{{space.app_count}}</td>
+                        <td>{{space.mem_dev_total}}</td>
+                        <td>{{space.mem_prod_total}}</td>
+                    </tr>
+                </tbody>
+            </table>
+        </div>
+        <div class="text-center" ng-show="spaces.length == 0">
+            <h2>This org has no spaces</h2>
```
Comment (jcscottiii) [inline static/js/app/partials/organizations.html]: org -> organization

File: static/js/app/partials/organizations.html (hunk 4 above)
Comment (ramirezg) [inline static/js/app/partials/organizations.html]: let's use ng-cloak

File: static/js/app/partials/organizations.html (hunk 5 above)
Comment (jcscottiii) [inline static/js/app/partials/organizations.html]: org -> organization

File: static/js/app/controllers.js (hunk 3 above)
Comment (ramirezg) [inline static/js/app/controllers.js]: yep

**Comment by ramirezg** [comment]
@jcscottiii This should be done for now. The coverage on the front end dropped a little but we can add the small tests later.

**Pull request merged**